### DISCUSSION TITLE
Multiple rnodes running from one reticulum instance?(also, antenna diversity)

### DISCUSSION THREAD

**Discussion by pizzadoglover911** [markqvist/Reticulum#267] [Help & Questions] [answered]
As the title suggests, is it possible to transmit through two(or more) rnodes from one computer running reticulum? For example, to allow for two antenna types at one location?

Or could antenna diversity be achieved on the Rnode hardware and software?

Just some thoughts I've been pondering while trying to buy out of stock marked up single board computers...

**Comment by markqvist** [comment]
You can add as many RNode interfaces to any one given RNS instance as you want. They will act as independent interfaces, so it is probably not going to be helpful running them on the same channel configurations, unless you really know what you are doing.

**Reply by pizzadoglover911** [reply]
The main use I was going for, would be directional and Omni directional antenna at a base station, where power is available and range/coverage are both needed.

 I figure I would need two radios to perform the function. I suppose what you are saying is they would ideally be shifted on the frequency as to not interfere?

**Reply by markqvist** [reply, marked as answer]
Yes, that is the standard pattern employed when setting up a radio site like that. With some directional antennas, and very good isolation between them, you might be able to get away with running the same frequency on two directional ones, but in general, and unless you really know what you're doing, I'd really suggest avoiding it.

More advanced functionality, like slaving different units together to perform MIMO and antenna diversity on the same channel is coming in the future (but this requires the units coordinating directly over a fast bus for this to work).

**Reply by pizzadoglover911** [reply]
Thanks for the help. This has lead me to another thought. 

Would it be possible to have an Rnode operate independent of an attached reticulum device with the goal of simply repeating the Lora signal without any other interfaces? Low power more compact setup. Without anything to host reticulum I guess it's a moot point.

**Reply by Swissbandit** [reply]
Yes,but for this to happen we need a version of rns in microphyton
Then it will be much easier to have stand alone devices and less energy consumation and smaller repeaters.
